zst/object.go

```diff
@@ -10,20 +10,12 @@
 //
 // An Object provides the interface to the underlying storage object.
 // To generate rows or cuts (and in the future more sophisticated traversals
-// and introspection), an Assembly is created from the Object then zng records
-// are read from the assembly, which implements zio.Reader.  The Assembly
-// keeps track of where each column is, which is why you need a separate
-// Assembly per scan.
-//
-// You can have multiple Assembly's referring to one Object as once an
-// object is created, it's state never changes.  That said, each assembly
-// will issue reads to the underlying storage object and the read pattern
-// may create performance issues.
+// and introspection), a Metadata instance is created from the Object then zng records
```

Comment:
```suggestion
// and introspection), a Metadata instance is created from the Object and then Zed records
```

Reply:
Deleting this paragraph as it's no longer accurate.